---
title: "Single-cell multi-cohort dissection of the schizophrenia transcriptome"

authors:
- Brad W. Ruzicka
- Shahin Mohammadi
- John F Fullard
- Jose Davila-Velderrain
- Sivan Subburaju
- Daniel Reed Tso
- Makayla Hourihan
- Shan Jiang
- Hao-Chih Lee
- Jaroslav Bendl
- PsychENCODE Consortium
- Georgios Voloudakis
- Vahram Haroutunian
- Gabriel E Hoffman
- Panos Roussos
- Manolis Kellis


date: "2024/5/24"
doi: "10.1126/science.adg5136"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Science"
publication_short: ""

abstract: INTRODUCTION. Schizophrenia population genomics has identified strong germline genetic associations for this highly heritable disorder, and molecular investigation of postmortem brain samples has yielded evidence of transcriptomic and epigenomic alterations associated with this disease. However, identifying molecular and cellular pathophysiological processes linking etiological risk factors and clinical presentation remains a challenge, due in part to the complex cellular architecture of the brain -- RATIONALE. Past work has implicated specific populations of excitatory and inhibitory neurons in the pathophysiology of schizophrenia, but existing large transcriptomic datasets of bulk tissue samples cannot directly assess cell type–specific contributions to disease. Single-cell RNA sequencing technologies allow measurement of genome-wide gene expression in individual cells with high-throughput, moving beyond bulk tissue measures to map disease-associated transcriptional changes in discrete cellular populations without bias toward preselected cell types. Investigating disease-associated phenotypic changes across the myriad cellular populations of the human brain can produce new insights into neuropsychiatric disease biology -- RESULTS. Using multiplexed single-nucleus RNA sequencing, we developed a single-cell resolution transcriptomic atlas of the prefrontal cortex across subjects with and without schizophrenia and present data from 468,727 nuclei isolated from 140 individuals across two well-defined and independently assayed cohorts. We identified expression profiles of brain cell types and neuronal subpopulations and systematically characterized the transcriptional changes associated with schizophrenia in each. For completeness, we report independent, cohort-specific analyses and joint meta-analysis of differential expression across 25 cell types. Using these data, we identified highly cell type–specific and reproducible expression changes, with 6634 differential expression events affecting 2455 genes and favoring down-regulated gene expression within excitatory neuronal populations. We found significant overlap with previously reported bulk cortex expression changes, primarily for excitatory neuronal populations, whereas changes in lower-abundance cell types were less efficiently captured in tissue-level profiling. Differentially expressed genes enrich neurodevelopmental and synapse-related molecular pathways and point to a regulatory core of coexpressed transcription factors linked to genetic risk variants for schizophrenia and developmental delay. Transcription factor targeting of schizophrenia differentially expressed genes in neuronal populations was validated with CUT&Tag in neuronal nuclei isolated from human prefrontal cortex. Furthermore, both transcriptional changes and putative upstream regulatory factors were enriched with genes harboring common and rare risk variants for schizophrenia, presenting evidence that genetic risk variants across the population frequency spectrum tend to target genes with measurable expression alterations in the excitatory neurons of patients with schizophrenia. Finally, the magnitude of schizophrenia-associated transcriptomic change segregated two populations of schizophrenia subjects. Transcriptomic heterogeneity within the cohorts was associated with specific cellular states shared across multiple neuronal populations, marked by genes related to synaptic function and one-carbon metabolism, suggesting genes characterizing distinct molecular phenotypes of schizophrenia.


# Summary. An optional shortened abstract.
summary: Our results provide a valuable resource to investigate the molecular pathophysiology of schizophrenia at single-cell resolution, offering insights into preferential dysregulation of specific neuronal populations and their potential role in mediating genetic risk. Together, they suggest convergence of etiological genetic risk factors, neuronal transcriptional dysregulation, and symptomatic manifestation in schizophrenia.

tags:
- Source Themes
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: ''
url_code: ''
url_dataset: 'https://www.psychencode.org/'
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

---
