---
title: "Single-cell atlas of transcriptomic vulnerability across multiple neurodegenerative and neuropsychiatric diseases"

authors:
- Donghoon Lee
- Mikaela Koutrouli
- Nicolas Y Masse
- Gabriel E Hoffman
- Seon Kinrot
- Xinyi Wang
- NM Prashant
- Milos Pjanic
- Tereza Clarence
- Fotios Tsetsos
- Deepika Mathur
- David Burstein
- Karen Therrien
- Aram Hong
- Clara Casey
- Zhiping Shao
- Marcela Alvia
- Stathis Argyriou
- Jennifer Monteiro Fortes
- Pavel Katsel
- Pavan K Auluck
- Lisa L Barnes
- Stefano Marenco
- David A Bennett
- PsychAD Consortium
- Lars Juhl Jensen
- Kiran Girdhar
- Georgios Voloudakis
- Vahram Haroutunian
- Jaroslav Bendl
- John F Fullard
- Panos Roussos

date: "2024-11-04"
doi: "10.1101/2024.10.31.24316513"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["3"]

# Publication name and optional abbreviated publication name.
publication: "medRxiv"
publication_short: ""

abstract: Neurodegenerative and neuropsychiatric diseases impose a significant societal and public health burden. However, our understanding of the molecular mechanisms underlying these highly complex conditions remains limited. To gain deeper insights into the etiology of different brain diseases, we used specimens from 1,494 unique donors to generate a population-scale single-cell transcriptomic atlas of the human dorsolateral prefrontal cortex (DLPFC), comprising over 6.3 million individual nuclei. The cohort includes neurotypical controls as well as donors affected by eight common and complex brain disorders, i.e. Alzheimer’s disease (AD), diffuse Lewy body disease (DLBD), vascular dementia (Vas), Parkinson’s disease (PD), tauopathy, frontotemporal dementia, schizophrenia, and bipolar disorder. We show that inter-individual variation accounts for a substantial portion of gene expression variation in the DLPFC. By comparing transcriptomic variation across diseases, we reveal universal signatures enriched in basic cellular functions such as mRNA splicing and protein localization. After discounting these cross-disease signatures, we show strong genetic and transcriptomic concordance among AD, DLBD, Vas, and PD, largely driven by alteration of synaptic signaling functions in neurons. Furthermore, we characterize transcriptomic variation among different AD phenotypes that were distinct from healthy aging. We uncover mitigating effects of interneurons and aggravating effects of immune and vascular cells in AD dementia. Further exploring the effect of the neuropsychiatric symptoms frequently accompanying AD, we identify a link to deep layer excitatory neurons. By constructing transcriptome trajectories that capture AD progression, we show cell-type specific responses implicated in early and late stages of AD. Our atlas provides an unprecedented perspective of the transcriptomic landscape in neurodegenerative and neuropsychiatric diseases, shedding light on shared and distinct processes involving the neuro-immune-vascular systems, and identifying potential targets for therapeutic intervention.

# Summary. An optional shortened abstract.
summary: We generated a single-cell transcriptomic atlas of 6.3 million nuclei from 1,494 donors spanning eight brain disorders and controls, revealing shared and distinct cellular and molecular signatures across neurodegenerative and neuropsychiatric diseases, with insights into disease-specific mechanisms, progression, and therapeutic targets.


tags:
- Source Themes
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

---
