---
title: "An atlas of chromatin accessibility in the adult human brain"

authors:
- John F. Fullard*
- Mads E. Hauberg*
- Jaroslav Bendl
- Gabor Egervari
- Maria-Daniela Cirnaru
- Sarah M. Reach
- Jan Motl
- Michelle E. Ehrlich
- Yasmin L. Hurd
- Panos Roussos

date: "2018-06-26"
doi: "10.1101/gr.232488.117"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "Genome Research"
publication_short: ""

abstract: Most common genetic risk variants associated with neuropsychiatric disease are noncoding and are thought to exert their effects by disrupting the function of cis regulatory elements (CREs), including promoters and enhancers. Within each cell, chromatin is arranged in specific patterns to expose the repertoire of CREs required for optimal spatiotemporal regulation of gene expression. To further understand the complex mechanisms that modulate transcription in the brain, we used frozen postmortem samples to generate the largest human brain and cell-type–specific open chromatin data set to date. Using the Assay for Transposase Accessible Chromatin followed by sequencing (ATAC-seq), we created maps of chromatin accessibility in two cell types (neurons and non-neurons) across 14 distinct brain regions of five individuals. Chromatin structure varies markedly by cell type, with neuronal chromatin displaying higher regional variability than that of non-neurons. Among our findings is an open chromatin region (OCR) specific to neurons of the striatum. When placed in the mouse, a human sequence derived from this OCR recapitulates the cell type and regional expression pattern predicted by our ATAC-seq experiments. Furthermore, differentially accessible chromatin overlaps with the genetic architecture of neuropsychiatric traits and identifies differences in molecular pathways and biological functions. By leveraging transcription factor binding analysis, we identify protein-coding and long noncoding RNAs (lncRNAs) with cell-type and brain region specificity. Our data provide a valuable resource to the research community and we provide this human brain chromatin accessibility atlas as an online database “Brain Open Chromatin Atlas (BOCA)” to facilitate interpretation.

# Summary. An optional shortened abstract.
summary: Here, we generated ATAC-seq atlas from 14 brain regions and two cell types of human brains.

tags:
- Source Themes
featured: false

# links:
# - name: ""
#   url: ""
url_pdf: 'https://genome.cshlp.org/content/28/8/1243.full.pdf+html'
url_code: ''
url_dataset: 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96949'
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

---
