---
title: "PredictSNP: robust and accurate consensus classifier for prediction of disease-related mutations"
authors:
- admin
- Jan Stourac
- Ondrej Salanda
- Antonin Pavelka
- Eric D. Wieben
- Jaroslav Zendulka
- Jan Brezovsky
- Jiri Damborsky 
date: "2014-01-01"
doi: "10.1371/journal.pcbi.1003440"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "PLoS computational biology"
publication_short: ""

abstract: Single nucleotide variants represent a prevalent form of genetic variation. Mutations in the coding regions are frequently associated with the development of various genetic diseases. Computational tools for the prediction of the effects of mutations on protein function are very important for analysis of single nucleotide variants and their prioritization for experimental characterization. Many computational tools are already widely employed for this purpose. Unfortunately, their comparison and further improvement is hindered by large overlaps between the training datasets and benchmark datasets, which lead to biased and overly optimistic reported performances. In this study, we have constructed three independent datasets by removing all duplicities, inconsistencies and mutations previously used in the training of evaluated tools. The benchmark dataset containing over 43,000 mutations was employed for the unbiased evaluation of eight established prediction tools, i.e.,  MAPP, nsSNPAnalyzer, PANTHER, PhD-SNP, PolyPhen-1, PolyPhen-2, SIFT and SNAP. The six best performing tools were combined into a consensus classifier PredictSNP, resulting into significantly improved prediction performance, and at the same time returned results for all mutations, confirming that consensus prediction represents an accurate and robust alternative to the predictions delivered by individual tools. A user-friendly web interface enables easy access to all eight prediction tools, the consensus classifier PredictSNP and annotations from the Protein Mutant Database and the UniProt database. The web server and the datasets are freely available to the academic community at [http://loschmidt.chemi.muni.cz/predictsnp](http://loschmidt.chemi.muni.cz/predictsnp).

# Summary. An optional shortened abstract.
summary: General meta-predictor of the effect of amino acid substitution on the protein function (not limited to humans).

tags:
- Source Themes
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1003440&type=printable
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [**Unsplash**](https://unsplash.com/photos/jdD8gXaTZsc)'
  focal_point: ""
  preview_only: false

---
